src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+button {
+  outline: none;
+  border: none;
+}
+
+i {
+  opacity: 0.8;
+  color: gray;
+}
+
+.main-section {
+  background-color: #fff;
+  border-radius: 5px;
+  margin: 10% 10%;
+  box-shadow: 3px 3px 3px 2px gray;
+  padding: 2% 2%;
+}
+
+.title {
+  display: flex;
+  justify-content: space-between;
+  padding: 3% 0;
+  border-bottom: 1px solid gray;
+}
+
+.title h1 {
+  font-size: 1.2rem;
+}
+
+.title i {
+  display: block;
+}
+
+.form-div {
+  display: flex;
+  justify-content: space-between;
+  padding: 2% 0;
+  min-height: 3%;
+  border-bottom: 1px solid gray;
+}
+
+.form-input {
+  display: flex;
+  justify-content: space-between;
```

Comment:
Try to use DRY with your code. There are multiple lines of code repeated as "display: flex" and "justify-content: space-between"

Reply:
Yeah sure, thanks